src/Server/Remote/SshExtension.php

```diff
@@ -109,8 +109,12 @@ public function checkConnection()
     /**
      * {@inheritdoc}
      */
-    public function run($command)
+    public function run($command, $timeout = null)
     {
+        if ($timeout !== null) {
+            throw new \RuntimeException('Timeout is not supported in ssh connection.');
```

Comment:
It should just throw this exception, or what do you mean? Maybe the exception should have better message?